jobs/pull-test-infra-bazel.sh

```diff
@@ -26,35 +26,13 @@ pip install -r jenkins/test_history/requirements.txt
 # Cache location.
 export TEST_TMPDIR="/root/.cache/bazel"
 
-# Compute list of modified files in bazel package form.
-commit_range="${PULL_BASE_SHA}...${PULL_PULL_SHA}"
-files=(bazel query "set($(git diff --name-only --diff-filter=d "${commit_range}"))")
-if [[ "${#files[@]}" == 0 ]]; then
-  echo "No bazel packages affected."
-  exit 0
-fi
-
-# Build modified packages.
-buildables=$(bazel query \
-    --keep_going \
-    --noshow_progress \
-    "kind(.*_binary, rdeps(//..., set(${files[@]})))")
-rc=0
-if [[ ! -z "${buildables}" ]]; then
-  bazel build ${buildables} && rc=$? || rc=$?
-  # Clear test.xml so that we don't pick up old results.
+if [[ -d bazel-testlogs ]]; then
```

Comment:
I think the issue is that we share the bazel workspace with other test runs, so when the symlink gets created, it might link to an old run. @spxtr can confirm.